cdap-ui/app/cdap/components/DataPrep/Directives/MarkAsError/index.js

```diff
@@ -157,11 +152,9 @@ export default class MarkAsError extends Component {
             className="cursor-pointer"
             onClick={this.toggleIgnoreCase}
           >
-            <span
-              className={classnames('fa', {
-                'fa-square-o': !this.state.ignoreCase,
-                'fa-check-square': this.state.ignoreCase
-              })}
+            <IconSVG
+              className="fa"
+              name={this.state.ignoreCase ? "icon-check-square-o" : "icon-square-o"}
```

Comment:
While converting to SVG you changed the `check-square` icon to `check-square-o` icon. Was this intentional? Right now in DataPrep the pattern is to use `check-square`